pootle/apps/virtualfolder/helpers.py

```diff
@@ -46,3 +48,15 @@ def extract_vfolder_from_path(request_path):
         return None, request_path
     else:
         return vfti.vfolder, vfti.directory.pootle_path
+
+
+def vftis_for_children(directory):
+    """
+    Returns the vfoldertreeitems for a directory's children
+    """
+    child_pks = [
+        child.pk
+        for child
+        in directory.children
+        if isinstance(child, Directory)]
+    return VirtualFolderTreeItem.objects.filter(directory__pk__in=child_pks)
```

Comment:
As long as it is not just `child` or `children` which also comprise stores, works for me.